wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -59,8 +59,9 @@
 @register.inclusion_tag("wagtailadmin/shared/breadcrumb.html", takes_context=True)
 def explorer_breadcrumb(
     context,
-    page,
     url_name,
+    page=None,
+    page_to_move=None,
```

Comment:
as discussed - you should no longer need to make changes to this template tag but it is good to note that the `page` being required was intentional so that you get an error if you do not give it a page.